yaspv2.js

```diff
@@ -33,25 +35,23 @@ import
 }
 from 'react-router-redux';
 // Load CSS
+
 require('./node_modules/font-awesome/css/font-awesome.css');
-//require('./node_modules/dota2-minimap-hero-sprites/assets/stylesheets/dota2minimapheroes.css');
```

Comment:
we're going to need the sprites back at some point (currently used on teamfight death map)